.github/CODEOWNERS

```diff
@@ -188,6 +188,7 @@
 /adaptive_shield/                        @adaptiveshield @DataDog/ecosystems-review
 /typingdna_activelock/                   @TypingDNA/logintegrations @raulpopa @endresstefan @DataDog/ecosystems-review
 /sofy_sofy/                              @Sofy @DataDog/ecosystems-review
+/notion/                                 @makenotion/public-api
 
 # Community Partners + Documentation
 /1e/manifest.json                        support@1e.com @DataDog/documentation @DataDog/ecosystems-review
```

Comment:
Can you also please add entries for `notion` in this `# Community Partners + Documentation` section (see `neoload` as an example and make sure to add `@DataDog/ecosystems-review` to each line)